scripts/build.bat

```diff
@@ -0,0 +1,14 @@
+@echo off
+title build
+
+:: Navigate to the folder of this script.
+cd %cd%
```

Comment:
`%cd%` refers to the current working directory, rather than the directory containing the script itself. This makes it impossible to run it from the root directory of the project, as the script will navigate to the directory above it. Instead, we should use `cd %~dp0`, this will ensure the correct script directory is selected.

Reply:
That's absolutely correct! I made a mistake there.

If it is executed in the working directory, we get the following results:
```
%~dp0: D:\GitHub\dirplayer-rs\scripts\
%cd%: D:\GitHub\dirplayer-rs
```